server/emitImports.js

```diff
@@ -8,7 +8,7 @@
 
 */
 
-import glob from 'glob'
+import * as glob from 'glob'
```

Comment:
@aacic when I tried `npm run dev` in `sjpp`, I got this error 
```sh
$ npm run dev

> @sjcrh/sjpp@1.0.0 dev
> trap 'kill 0' SIGINT; npm run ppbundle & npm start


> @sjcrh/sjpp@1.0.0 ppbundle
> cd proteinpaint; npm run bundle; cd ..


> @sjcrh/sjpp@1.0.0 prestart
> tsx proteinpaint/server/emitImports.js dev > ./server.ts


> @sjcrh/proteinpaint@2.52.0 bundle
> cd client; webpack --env NODE_ENV=development --watch --color; cd ..

file:///Users/esioson/dev/sjpp/proteinpaint/server/emitImports.js:32
			const files = glob.sync('*.ts', { cwd })
			                   ^

TypeError: glob.sync is not a function
    at file:///Users/esioson/dev/sjpp/proteinpaint/server/emitImports.js:32:23
    at ModuleJob.run (node:internal/modules/esm/module_job:218:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:329:24)
    at async loadESM (node:internal/process/esm_loader:28:7)
    at async handleMainPromise (node:internal/modules/run_main:113:12)

```

Reply:
yes, and the tests failed. 

It works when I change it to:

const glob = await import('glob');

I've pushed again.